### PR TITLE
11.0 merge

### PR DESCRIPTION
**Pull request by Ommo73**

### CODE REVIEW

File: pos_cashier_select/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.0.2`
+-------
+
+**Fix:** Incompatibility with pos_scan_ref module
+
```
Comment (kaadevelop) [inline pos_cashier_select/doc/changelog.rst:5]: Has to be tested

File: pos_keyboard/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+`1.1.5`
+-------
+
+**Fix:** Function 'disconnect' not working in expected way
+
+`1.1.4`
+-------
+
+**Fix:** Incompatibility with some modules led to the action duplicating
+
```
Comment (kaadevelop) [inline pos_keyboard/doc/changelog.rst:11]: Has to be tested

File: pos_longpolling/doc/changelog.rst [hunk 3]
```diff
@@ -1,3 +1,8 @@
+`2.2.0`
+-------
+
+- **IMP:** *Autostart longpolling* field is hidden on the `POS config` form to prevent possible misconfigurations.
+
```
Comment (kaadevelop) [inline pos_longpolling/doc/changelog.rst:5]: Has to be tested

File: pos_multi_session/doc/changelog.rst [hunk 4]
```diff
@@ -1,3 +1,8 @@
+`4.2.9`
+-------
+
+- **Fix:** Error "journal_id is not defined" after add new journal to another POS with same multi-session
+
```
Comment (kaadevelop) [inline pos_multi_session/doc/changelog.rst:5]: Has to be tested

**Review by kaadevelop — COMMENTED** [review]
**8 installable** modules are updated:
```
├─ pos_cashier_select/
|  ├─ __init__.py
|  ├─ __manifest__.py
|  ├─ doc/
|  |  └─ changelog.rst
|  ├─ static/
|  |  └─ src/
|  |     ├─ js/
|  |     |  ├─ main.js
|  |     |  └─ tour.js
|  |     └─ xml/
|  |        └─ pos.xml
|  ├─ tests/
|  |  ├─ __init__.py
|  |  └─ test_default.py
|  └─ views/
|     └─ views.xml
├─ pos_journal_pin/
|  └─ static/
|     └─ src/
|        └─ js/
|           └─ test_pos_journal_pin.js
├─ pos_keyboard/
|  ├─ __manifest__.py
|  ├─ doc/
|  |  └─ changelog.rst
|  ├─ static/
|  |  └─ src/
|  |     └─ js/
|  |        ├─ pos.js
|  |        └─ test_pos_keyboard.js
|  └─ tests/
|     └─ test_pos_keyboard.py
├─ pos_longpolling/
|  ├─ __manifest__.py
|  └─ doc/
|     └─ changelog.rst
├─ pos_multi_session/
|  ├─ __manifest__.py
|  ├─ doc/
|  |  └─ changelog.rst
|  ├─ static/
|  |  └─ src/
|  |     └─ js/
|  |        └─ pos_multi_session.js
|  └─ views/
|     └─ pos_multi_session_views.xml
├─ pos_multi_session_restaurant/
|  ├─ README.rst
|  ├─ doc/
|  |  └─ index.rst
|  └─ static/
|     └─ src/
|        └─ js/
|           └─ pos_multi_session_restaurant.js
├─ pos_multi_session_sync/
|  ├─ __manifest__.py
|  ├─ doc/
|  |  └─ changelog.rst
|  └─ models/
|     └─ pos_multi_session_sync_models.py
└─ pos_scan_ref/
   ├─ __manifest__.py
   ├─ static/
   |  └─ src/
   |     └─ js/
   |        └─ test_pos_scan_ref.js
   └─ tests/
      └─ test_pos_scan_ref.py
```

**1 not installable** modules  is updated:
```
└─ pos_product_lot/
   ├─ README.rst
   ├─ __manifest__.py
   ├─ doc/
   |  └─ changelog.rst
   └─ static/
      └─ src/
         └─ js/
            └─ pos.js
```

> sent by [:v: Review Bot](https://odoo-devops.readthedocs.io/en/latest/git/github-review-bot.html)

File: pos_multi_session_sync/doc/changelog.rst [hunk 5]
```diff
@@ -1,3 +1,8 @@
+`1.0.5`
+-------
+- **Fix:** Cross multi-sesion messaging in case the same pos was assigned to a different multi-session
+- **Fix:** Possible loss of orders for multiple multi-sessions working simultaneously when another new one starts
+
```
Comment (kaadevelop) [inline pos_multi_session_sync/doc/changelog.rst:5]: Has to be tested

Comment (Ramil-Mukhametzyanov) [reply pos_multi_session_sync/doc/changelog.rst:5]: >> **Fix:** Possible loss of orders for multiple multi-sessions working simultaneously when another new one starts
:+1: checked

**Review by ImplantiSolucoes — COMMENTED** [review]
Nice

**Pull request closed without merging**